server/db/models/cocktail.js

```diff
@@ -0,0 +1,30 @@
+const Sequelize = require('sequelize')
+const db = require('../db')
+const axios = require('axios');
+
+
+const Cocktail = db.define('cocktail', {
+    name: {
+        type: Sequelize.STRING,
+        allowNull: false
+    },
+    imageUrl: {
+        type: Sequelize.STRING,
+        defaultValue: 'https://upload.wikimedia.org/wikipedia/commons/thumb/e/e7/Flaming_cocktails.jpg/220px-Flaming_cocktails.jpg'
+    }
+    // idCoctail: {
```

Comment:
i would completely remove idCocktail, then uncomment ingredients and price? maybe allow them to be null so it doesn't mess up, but we will eventually need